packages/mobile/src/screens/profile-screen/utils.ts

```diff
@@ -64,6 +63,6 @@ export const useShouldShowCollectiblesTab = () => {
  * trims both leading and trailing newlines
  * @param {string} str
  */
-export const squashNewLines = (str: Nullable<string>) => {
-  return str ? str.replace(/\n\s*\n\s*\n/g, '\n\n').trim() : str
+export const squashNewLines = (str?: string) => {
+  return str ? str.replace(/\n\s*\n\s*\n/g, '\n\n').trim() : ''
```

Comment:
We're sure this change doesn't affect anything elsewhere, right?

Reply:
good q i think not, since any caller that originally used it had to pass a string, this just helps the one case where bio may be undefined so caller doesn't need to ?? ''